### ISSUE TITLE
Stabilizing the team1 branch (part 1)

### ISSUE THREAD

**Issue by hbarri** [rsanchez-wsu/jfiles#340]
In this issue, we plan on continuing our merge process of team1 and team1-dev. We plan to create two new branches from commits 22169e1 and a987e87 and then merge those two branches together to hopefully be able to merge that commit with commit f62c3ca later on.

**Comment by hbarri** [comment]
Jason (@jasonmarcum) and I were able to create the two new branches from commits 22169e1 and a987e87 (which we called team1-dev2 and team1-dev3) and start the merging process. However, because the team1 and team1-dev branches have diverged so significantly, we were only able to manually complete the merge of the client side but have not gotten to the server class. We plan to finish this up in the next milestone.
